### ISSUE TITLE
queue popup

### ISSUE THREAD

**Issue by amsterr05** [TeamFightTacticsBots/Alune#11]
hi, bot works like a charm but there is an ongoing problem with mobile version being different than on pc, whenever bot accepts match this thing pops up and bot is stuck on it and you have to manually accept that it is okay to play. I followed all instructions etc tried updating from the store but nothing works and the popup is still there, I believe its on riot side because on my phone I have the same problem

![Screenshot_11](https://github.com/TeamFightTacticsBots/Alune/assets/169453820/34de7edc-5584-4b5f-960e-38eba57566e4)

**Comment by RomeoAlphaEkko** [comment]
Seems you have an old game version 

This is TFT side-ly not the bot

**Comment by amsterr05** [comment]
I did it like in instructions and it's still there

downloaded form this link, version from 1 may 14.9.5798615 because its latest 
https://www.apkmirror.com/apk/riot-games-inc/teamfight-tactics-league-of-legends-strategy-game/

**Comment by RomeoAlphaEkko** [comment]
you are correct that is the newest version, the other question should be if it wants to update when starting the app, u did it right? sorry for asking stupid questions, just want to cross out problem reasons

**Comment by amsterr05** [comment]
yes it asked if I want to patch the game, had to did it otherwise it wouldn't start. 
I think it's a tft sided problem because on my own phone I have the same popup whenever I accept the que despite being the latest too

**Comment by RomeoAlphaEkko** [comment]
> If the app is outdated, the bot will refuse to start, so there is no newer APK you could download. I assume it takes time for Google Play to propagate app versions (and for APKMirror to mirror it). We'll need to add a check for this, so it at least clicks the check mark.

![image](https://github.com/TeamFightTacticsBots/Alune/assets/89660477/9bf528b4-202f-4c0f-87e8-14c72cc216b8)
is the working version i have, and if i didnt miss the numbers they are the same

**Comment by akshualy** [comment]
An untested fix is available when running from source, I will make a release once I had the chance to see this fix in action.
